src/engine/enginebuffer.cpp

```diff
@@ -375,9 +374,8 @@ void EngineBuffer::queueNewPlaypos(double newpos, enum SeekRequest seekType) {
         // use SEEK_STANDARD for that
         seekType = SEEK_STANDARD;
     }
-    m_queuedSeekPosition.setValue(newpos);
-    // set m_queuedPosition valid
-    m_iSeekQueued = seekType;
+    struct QueuedSeek queuedSeek = {newpos, seekType};
```

Comment:
More idiomatic C++:
const auto queuedSeek = QueuedSeek{newpos, seekType};

The temporary only introduces redundant noise, I would prefer:
m_queuedSeek.setValue(QueuedSeek{newpos, seekType});